theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from theano.tensor.nnet.abstract_conv import get_conv_output_shape
+
+
+class TestGetConvOutShape(unittest.TestCase):
+
+    def test_basic(self):
+        image_shape, kernel_shape = (3, 2, 8, 8), (4, 2, 5, 5)
+        sub_sample = (1, 2)
+        test1_params = get_conv_output_shape(
+            image_shape, kernel_shape, 'valid', sub_sample)
+        test2_params = get_conv_output_shape(
+            image_shape, kernel_shape, 'half', sub_sample)
+        test3_params = get_conv_output_shape(
+            image_shape, kernel_shape, 'full', sub_sample)
```

Comment:
To increase test coverage, add a test case when the 3rd parameter is a tuple of int and a test case with just an int.

With both of those change, I think it is good. thanks